internal/store/files/recog_place_store.go

```diff
@@ -77,10 +79,16 @@ type RecogPlaceStore struct {
 
 // LoadRecogPlaceStore loads RecogPlaceStore.
 func LoadRecogPlaceStore() (*RecogPlaceStore, error) {
-	reader := csv.NewReader(strings.NewReader(string(recogPlaceMapCSVContent)))
-	records, err := reader.ReadAll()
-	if err != nil {
-		return nil, err
+	records := [][]string{}
+	for _, fileContent := range [][]byte{recogPlaceMapCSVContent, recogEntityMapCSVContent} {
+		reader := csv.NewReader(strings.NewReader(string(fileContent)))
+		fileRecords, err := reader.ReadAll()
+		if err != nil {
+			return nil, err
+		}
+		// Remove the header
+		fileRecords = fileRecords[1:]
+		records = append(records, fileRecords...)
```

Comment:
Can we just do:  `records = append(records, fileRecords[1:]...)` ?

Probably doesn't matter, just in case `fileRecords = fileRecords[1:]` translates to mem copies...

Reply:
done